Detectors/CTF/workflow/src/CTFWriterSpec.cxx

```diff
@@ -264,12 +265,22 @@ size_t CTFWriterSpec::processDet(o2::framework::ProcessingContext& pc, DetID det
     for (int ib = 0; ib < C::getNBlocks(); ib++) {
```

Comment:
Once a given block has overflown once, there is no point in continuing accumulation for it at producing warnings.
Could you add:
```
static uint64_t overflowStatus = 0;
for (int ib = 0; ib < C::getNBlocks(); ib++) {
  if (overflowStatus & (0x1UL<<ib)) {
     continue;
  }
```
and set in the catch after the warning 
```
overflowStatus |= (0x1UL<<ib);
```

Reply:
OK, I wanted to leave the benefit of doubt in case one tf overflows yet another does not. But if you prefer, I can change it.

Reply:
TFs should be rather uniform

Reply:
If that is the case I'll change that.